### DISCUSSION TITLE
Add LinkedIn link to profile

### DISCUSSION THREAD

**Discussion by opuu** [community/community#44204] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Can we have a LinkedIn social link in our profile next to twitter? 

<img width="155" alt="image" src="https://user-images.githubusercontent.com/77793409/212297299-769b17a6-4186-4d81-81ce-41e1a081dd9a.png">

**Comment by radar-blip** [comment, marked as answer]
In your settings

![socialaccts](https://user-images.githubusercontent.com/39196767/216235402-3f393bdb-b1f6-487d-ba92-675ef71afef0.png)

**Reply by opuu** [reply]
Was it there the whole time?! Never noticed it. Thanks.

**Reply by radar-blip** [reply]
No. I think it was just added very recently, like within the last 24 hours.
